crates/hotshot/src/traits/networking/web_server_network.rs

```diff
@@ -1275,6 +1277,8 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>, MEMBERSHIP: Membership<TYPES
             network_id,
             da_committee_size,
             is_da,
+            // unsupported
```

Comment:
Does this `unsupported` means `reliability_config` is unsupported? Maybe can add more comments here.

Reply:
Yep, exactly. I'll add more information to comments. TLDR is network reliability is only implemented for libp2p and memory_network right now